docker/readmodel-db/attribute.sql

```diff
@@ -7,7 +7,11 @@ CREATE TABLE IF NOT EXISTS readmodel_attribute.attribute (
   kind VARCHAR NOT NULL,
   description VARCHAR NOT NULL,
   origin VARCHAR,
-  name VARCHAR NOT NULL,
+  name VARCHAR NOT NULL UNIQUE,
```

Comment:
note: this constraint may not be applicable on deployed databases because existing attributes may be already duplicated